libs/shared/assistant/feature/src/lib/assistant-trigger/assistant-trigger.tsx

```diff
@@ -9,20 +12,39 @@ export interface AssistantTriggerProps {
 
 export function AssistantTrigger({ defaultOpen = false }: AssistantTriggerProps) {
   const { assistantOpen, setAssistantOpen } = useContext(AssistantContext)
+  const [assistantIcon] = useLocalStorage<AssistantIcon>(AssistantIconKey, 'QUESTION_MARK')
 
   return (
     <>
-      <button
-        className="group fixed bottom-8 right-8 flex h-14 w-14 items-center justify-center overflow-hidden rounded-full bg-brand-500 text-neutral-50 shadow-2xl outline-brand-600 transition hover:animate-[showAssistantTrigger_0.2s_cubic-bezier(0.21,1.02,0.73,1)_forwards] hover:bg-brand-600"
-        onClick={() => setAssistantOpen(!assistantOpen)}
-      >
-        <Icon
-          iconStyle="light"
-          iconName="question"
-          className="text-2xl group-hover:animate-[shake_0.6s_cubic-bezier(0.36,0.07,0.19,0.97)_both]"
-        />
-        <span className="pointer-events-none absolute left-0 top-0 h-[150px] w-[150px] bg-gradient-to-b from-transparent from-0% via-neutral-50 via-50% to-transparent to-100% opacity-0 group-hover:animate-[shineAssistantTrigger_1s_linear_forwards] group-hover:opacity-20"></span>
-      </button>
+      {match(assistantIcon)
+        .with('QUESTION_MARK', () => (
+          <button
```

Comment:
Could you create a component to avoid duplicate this DOM?

Reply:
I've factorize the code without creating a new component
https://github.com/Qovery/console/pull/1571/commits/e2777853b02a92e167237fcc75e35c6f29760c0c